### PR TITLE
Add a feature for admins to add new admins

### PR DESCRIPTION
**Pull request by sohan2410**
- Added /addadmin route in api/controllers/users/router where admin can admins by following middlewares
isAdmin and isAuthenticated

### CODE REVIEW

File: server/api/controllers/admin/controllers/users/router.js [hunk 1]
```diff
@@ -26,5 +26,6 @@ const upload = multer({
 
 export default express
   .Router()
-  .post("/upload", upload.single("users"), controller.uploadUsers)
-  .put("/:roll", controller.editUserDetails);
+  .post('/upload', upload.single('users'), controller.uploadUsers)
+  .put('/:roll', controller.editUserDetails)
+  .post('/addadmin', controller.addAdmin);
```
Comment (GunaShekar02) [inline server/api/controllers/admin/controllers/users/router.js:31]: We can rename this to `addAdmin` instead of `addadmin` to improve visibility.

**Review by GunaShekar02 — CHANGES_REQUESTED** [review]
This is really good work, thank you @sohan2410! I left a few minor comments. Other than this, I request you to please also write the documentation for this API. You can find the documentation in the [`api.yml`](https://github.com/AASF-IIITM/aasf-website-backend/blob/master/server/common/api.yml) file. It's very easy to write the documentation, you can take reference from [one of the other APIs](https://github.com/AASF-IIITM/aasf-website-backend/blob/master/server/common/api.yml#L308).

**Comment by sohan2410** [comment]
As mentioned I have added api documentation, description function in addAdmin and camelCase for variables.

File: server/common/api.yml [hunk 2]
```diff
@@ -927,3 +957,12 @@ components:
...
           description: Base64 encoded data URL of the image
+
+    AddAdmin:
+      type: object
+      required:
+        - userId
+      properties:
+        userId:
+          type: string
+          example: 2020IMG016
```
Comment (GunaShekar02) [inline server/common/api.yml:968]: The roll number format we use is `2020IMG-016`. Could you please change this?

**Review by GunaShekar02 — COMMENTED** [review]
Thanks for adding the documentation, there is just one very minor change to be done, I left a change for that. After that, we're good to go!

**Review by GunaShekar02 — APPROVED** [review]

**Pull request merged**